extensions/wikia/CuratedContent/CuratedContentHooks.class.php

```diff
@@ -74,7 +74,7 @@ private static function isAllowedWikia() {
 		);
 	}
 
-	private static function shouldDisplayCuratedContentToolButton() {
+	public static function shouldDisplayCuratedContentToolButton() {
```

Comment:
is hook a good place for this method? given we have CuratadContentHelper which is a bag of stuff maybe creating another one here is not the best idea